src/web/screens/clientSideIntegration.tsx

```diff
@@ -46,6 +53,28 @@ function ClientSideIntegration() {
     }
   };
 
+  const handleUpdateKeyPair = async (formData: UpdateKeyPairFormProps) => {
+    const { name, subscriptionId, disabled = false } = formData;
+    try {
+      const response = await UpdateKeyPair({ name, subscriptionId, disabled });
+      if (response.status === 201) {
```

Comment:
You're not checking for a 200 status in handleDisableKeyPair below.  I wonder if you even need to here.  Is it possible to return a 200 from here but not a 201?  If not you can just not check for it because a non-20x response will trigger the fallback handler and catch block.